pytentiostat/executable.py

```diff
@@ -0,0 +1,20 @@
+# This is what you want to execute
+import json
+from ArduinoBoard import ProgramSetup as PS
+#from DataHandeling import DataHandeling as DH
+
+
+
+if __name__ == '__main__':
+    test = PS()
+    #test.setup_arduino()
+    with open('config.json') as json_data_file:
```

Comment:
I would not call this file "data" since it doesn't contain data.  It will confuse people maybe.  I would do it:
```
i = open('config.json','r')
config = json.load(i)
```
and so on.